NAMESPACE

```diff
@@ -4,11 +4,9 @@ export("RcppMLPACK.package.skeleton",
        "lars",
        "pca",
        "serialize_gaussian_kernel_to_xml",
-       "serialize_gaussian_kernel",
-       "unserialize_gaussian_kernel",
        "serialize_lars_to_xml",
```

Comment:
Sorry, only noticing this now.  Maybe in the next roumd you can standardize multiple `serialize_X_to_xml` into a single `serialize_to_xml` ?

Reply:
Yeah, I will open a PR to standardize multiple `serialize_X_to_xml` into a single `serialize_to_xml` and to discussion more on this [function](https://github.com/Yashwants19/RcppMLPACK/blob/master/R/transform.R).